be/src/exec/vectorized/hdfs_scan_node.cpp

```diff
@@ -100,10 +99,25 @@ Status HdfsScanNode::prepare(RuntimeState* state) {
     }
 
     RETURN_IF_ERROR(ScanNode::prepare(state));
+
+    if (_hdfs_scan_node.__isset.hive_table_name) {
+        _runtime_profile->add_info_string("Table", _hdfs_scan_node.hive_table_name);
+    }
+    if (_hdfs_scan_node.__isset.sql_predicates) {
+        _runtime_profile->add_info_string("Predicates", _hdfs_scan_node.sql_predicates);
+    }
+    if (_hdfs_scan_node.__isset.min_max_sql_predicates) {
+        _runtime_profile->add_info_string("PredicatesMinMax", _hdfs_scan_node.min_max_sql_predicates);
+    }
+    if (_hdfs_scan_node.__isset.partition_sql_predicates) {
+        _runtime_profile->add_info_string("PredicatesPartition", _hdfs_scan_node.partition_sql_predicates);
+    }
```

Comment:
IMO, you can do this in `init` function, then the _hdfs_scan_node can be eliminated from this struct.